### ISSUE TITLE
[ui5-card][A11y]: Title refrence is not valid

### ISSUE THREAD

**Issue by GerganaKremenska** [SAP/ui5-webcomponents#3643]
### **Bug Description**
Refrence in the <ui5-card> pointing to the title inside the <ui5-card-header> is not correct.  Even if it is it will not point to the shadow root of the <ui5-card-header> and it will not be valid id again and the Screen readeer will not read it out

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/Card/
2. Turn on SR

Expected Behavior:
The title is read out. valid ids

Actual behavior:
The title is not read out. not valid ids

### **Log Output / Stack Trace / Screenshots**
```
{...}
```
### **Priority**
- [ ] Low
- [ ] Medium
- [x] High
- [ ] Very High

**Note:** 
Title of the card should be added as aria-label so it is read out
Remove the title-text id refrence from this function.
 get ariaLabelledByCard() {
        if (this.hasHeader) {
            labels = this.header[0].hasAttribute("title-text") ? `${this._id}--header-title ${this._id}-desc` : `${this._id}-desc`;
        }
        return labels;
}
